draft-ietf-trans-gossip.md

```diff
@@ -280,14 +285,18 @@ be unrevoked (which will be hard for a MitM to sustain)\]
 Refer to {{pooling-policy-recommendations}} for recommendations about
 strategies.
 
-An SCT MUST NOT be sent to any other HTTPS server than one serving the
-domain to which the certificate signed by the SCT refers. Not
+\[TODO: The above sentences that talk about the algorithm will be updated with the pooling recommendation section \]
+
+An SCT MUST NOT be sent to any other HTTPS server than one which 
+is an exact match for the domain the SCT was originally received under. Not
 following this constraint would lead to two types of privacy
 leaks. First, the server receiving the SCT would learn about other
-sites visited by the HTTPS client. Secondly, auditors or monitors
+sites visited by the HTTPS client, as explained above. Secondly, auditors or monitors
 receiving SCTs from the HTTPS server would learn information about the
 other HTTPS servers visited by its clients.
 
+\[TODO: this MUST is a repeat of the above. Do we need to re-iterate this? Probably not, so we can remove this paragraph? \]
+
```

Comment:
Move the rationale (starting with "Not following this...") upwards and remove the MUST?